extension/src/utils/cpUtils.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as cp from 'child_process';
+import * as kill from 'tree-kill';
+
+export function killProcess(process: cp.ChildProcess): Promise<void> {
+    return new Promise<void>((resolve: () => void, reject: (err: Error) => void): void => {
+        if (process) {
+            kill(process.pid, 'SIGTERM', (error: Error | undefined) => {
+                if (error) {
+                    reject(error);
```

Comment:
The code here doesn't have any problem, but it's better to return early after reject. 

See a post [here](https://stackoverflow.com/questions/32536049/do-i-need-to-return-after-early-resolve-reject)